<ui version="4.0" >
 <class>ToolsControl</class>
 <widget class="QDialog" name="ToolsControl" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>External Tools</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="3" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="test" >
       <property name="text" >
        <string>&amp;Test</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>349</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="1" column="2" >
      <widget class="QLineEdit" name="make" >
       <property name="readOnly" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="qmakeIcon" >
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="../resources/resources.qrc" >:/divers/images/good.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="7" column="3" >
      <widget class="QToolButton" name="designerLocation" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2" >
      <widget class="QLineEdit" name="ctags" >
       <property name="readOnly" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="1" >
      <widget class="QLabel" name="label_11" >
       <property name="text" >
        <string>lrelease:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <widget class="QLineEdit" name="qmake" >
       <property name="readOnly" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="3" >
      <widget class="QToolButton" name="linguistLocation" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3" >
      <widget class="QToolButton" name="makeLocation" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2" >
      <widget class="QLineEdit" name="lrelease" >
       <property name="readOnly" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="gdbIcon" >
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="../resources/resources.qrc" >:/divers/images/good.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLabel" name="label_6" >
       <property name="text" >
        <string>make:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="ctagsIcon" >
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="../resources/resources.qrc" >:/divers/images/good.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="6" column="3" >
      <widget class="QToolButton" name="lreleaseLocation" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" >
      <widget class="QLineEdit" name="gdb" >
       <property name="readOnly" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>qmake:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" >
      <widget class="QLabel" name="linguistIcon" >
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="../resources/resources.qrc" >:/divers/images/good.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="7" column="1" >
      <widget class="QLabel" name="label_12" >
       <property name="text" >
        <string>designer:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" >
      <widget class="QLabel" name="designerIcon" >
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="../resources/resources.qrc" >:/divers/images/good.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QLabel" name="label_8" >
       <property name="text" >
        <string>ctags:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3" >
      <widget class="QToolButton" name="qmakeLocation" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" >
      <widget class="QLabel" name="label_10" >
       <property name="text" >
        <string>lupdate:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3" >
      <widget class="QToolButton" name="lupdateLocation" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLabel" name="label_7" >
       <property name="text" >
        <string>gdb:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3" >
      <widget class="QToolButton" name="ctagsLocation" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2" >
      <widget class="QLineEdit" name="linguist" >
       <property name="readOnly" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0" >
      <widget class="QLabel" name="lupdateIcon" >
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="../resources/resources.qrc" >:/divers/images/good.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="2" column="3" >
      <widget class="QToolButton" name="gdbLocation" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2" >
      <widget class="QLineEdit" name="lupdate" >
       <property name="readOnly" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0" >
      <widget class="QLabel" name="lreleaseIcon" >
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="../resources/resources.qrc" >:/divers/images/good.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="7" column="2" >
      <widget class="QLineEdit" name="designer" >
       <property name="readOnly" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1" >
      <widget class="QLabel" name="label_9" >
       <property name="text" >
        <string>linguist:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="makeIcon" >
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="../resources/resources.qrc" >:/divers/images/good.png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Please enter the path to the external programs:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ToolsControl</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>356</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
